client/coral-sign-in/containers/SignInContainer.js

```diff
@@ -71,6 +75,17 @@ class SignInContainer extends Component {
     });
   }
 
+  handleChangeEmail(e) {
+    const {value} = e.target;
+    console.log('handleChangeEmail', value);
+    this.setState({emailToBeResent: value});
+  }
+
+  handleResendConfirmation(e) {
+    e.preventDefault();
+    this.props.requestConfirmEmail(this.state.emailToBeResent);
```

Comment:
This doesn't seem to give the user a confirmation or close the dialog. It should probably do one or both.